operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -175,6 +185,28 @@ The Technical Product Writer/s form part of the PR review/approval process. When
 
 The [Documentation Style guide](https://docs.mattermost.com/guides/core.html#documentation-style-guide) and [UI text guidelines](https://docs.mattermost.com/process/documentation-UItext-guidelines.html).
 
+## Customer Education Responsibilities
```

Comment:
```suggestion
## Customer Education Processes
```

Reply:
We can name the `process` section above this as `Technical Docs Processes`

Reply:
^distinguishing 'process' from 'responsibilities' listed for customer education above